react/admin/ReturnDetails/components/PickupAddress.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import { useReturnDetails } from '../../hooks/useReturnDetails'
+
+export const PickupAddress = () => {
+  const { data } = useReturnDetails()
+
+  if (!data) return null
+
+  const {
+    returnRequestDetails: {
+      pickupReturnData: { country, state, address, zipCode, city, addressType },
+    },
+  } = data
+
+  return (
+    <section className="flex-ns flex-wrap flex-auto flex-column pt4 pb4">
+      <h3>
+        <FormattedMessage id="admin/return-app.return-request-details.pickup-address.title" />
+      </h3>
+      <div className="mb5">
+        <p className="ma0">
+          <FormattedMessage id="admin/return-app.return-request-details.pickup-address.pickup-type" />
+          :{' '}
+          {addressType === 'PICKUP_POINT' ? (
+            <FormattedMessage id="admin/return-app.return-request-details.pickup-address.pickup-type.pickup-point" />
+          ) : (
+            <FormattedMessage id="admin/return-app.return-request-details.pickup-address.pickup-type.customer-address" />
+          )}
+        </p>
+      </div>
+      <div className="mb5">
+        <p className="ma0">
+          <FormattedMessage id="admin/return-app.return-request-details.pickup-address.country" />
+          : {country}
```

Comment:
You can encapsulate a dynamic value inside the FormattedMessage component. Take a look at this example [here](https://github.com/vtex-apps/return-app/blob/8b6fbd2ab82e59edcc7d5320d7c81136279c32a6/react/admin/settings/components/CustomReasons/TranslationsModal.tsx#L200-L207).